src/python/WMCore/WMSpec/StdSpecs/StepChain.py

```diff
@@ -421,16 +438,16 @@ def validateSchema(self, schema):
         Go over each step and make sure it matches validation parameters.
         """
         numSteps = schema['StepChain']
-        couchUrl = schema.get("ConfigCacheUrl", None) or schema["CouchURL"]
+        couchUrl = schema.get("ConfigCacheUrl", None) or schema.get("CouchURL", None)
```

Comment:
Actually, couchUrl cannot be None, since it's a mandatory parameter:
https://github.com/amaltaro/WMCore/blob/stepchain-intermediate/src/python/WMCore/WMSpec/StdSpecs/StepChain.py#L331

so in the end, the change in this line was useless.